recipes/r-pythoninr/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-pythoninr
+  version: "0.1_4"
+
+source:
+  fn: PythonInR_0.1-4.tar.gz
+  url: https://cran.r-project.org/src/contrib/PythonInR_0.1-4.tar.gz
+  sha256: ee9d5c33580b9273d80e43ed4d83dae81de2fc35cc92e310feefe5e4c12c394f 
+
+build:
+  number: 0
+  skip: true  # [py34]
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - python >=3.5
+    - r-base
+    - r-r6
+    - r-pack
+    - gcc
+  run:
+    - python >=3.5
+    - r-base
+    - r-r6
+    - r-pack
+    - gcc
+
+test:
+  commands:
+    - $R -e "library('PythonInR')"  # [not win]
+    - "\"%R%\" -e \"library('PythonInR')\""  # [win]
+
+about:
+  home: https://bitbucket.org/Floooo/pythoninr
+  license: GPL-3 
```

Comment:
Need license family and the license_file

Reply:
The [conda_r_skeleton_helper](https://github.com/bgruening/conda_r_skeleton_helper) will automatically handle the license file and also add some more maintainers.